### PR TITLE
feat: added text utils

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the text utilities.

image

image

image

### CODE REVIEW

**Comment by ashishkatiyar2404** [comment]
Comment some changes. 
your code good to me and UI also looks good💯

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Review by m26unkwn — APPROVED** [review]
Everything is looking good to me.
GTM

File: stylesheets/components/text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.p-grey {
+   color: grey;
+}
+
+.text-left{
+   text-align: left;
+}
+
+.text-center{
+   text-align: center;
```
Comment (pankaj-prog) [inline stylesheets/components/text-utilities/text-utilities.css:34]: Good naming, classes are descriptive.

**Review by pankaj-prog — APPROVED** [review]
Everything looks good , you can merge it

**Comment by SinchanaMS** [comment]
Merging to dev.

**Pull request merged**